From 5c578a23c9241c8a99ff2a1d22ec24d6b23ef92b Mon Sep 17 00:00:00 2001
From: Ananth Krishna R <ananth.krishna.r@intel.com>
Date: Fri, 20 Feb 2015 19:15:05 +0530
Subject: [PATCH] [MODULE LEVEL DVFS] Add Module DVFS configuration

The patch adds CONFIG_INTEL_MODULE_CPU_FREQ config

Change-Id: Id070616fee62342677462f304c7a0467176bf810
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-5644
Signed-off-by: Ananth Krishna R <ananth.krishna.r@intel.com>
Signed-off-by: Kumar P, Mahesh <mahesh.kumar.p@intel.com>
---
 drivers/cpufreq/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/cpufreq/Kconfig b/drivers/cpufreq/Kconfig
index f74f603..423c049 100644
--- a/drivers/cpufreq/Kconfig
+++ b/drivers/cpufreq/Kconfig
@@ -24,6 +24,16 @@ config CPU_FREQ_BOOST_SW
 	bool
 	depends on THERMAL
 
+config INTEL_MODULE_CPU_FREQ
+       bool "Intel module level cpu frequency scaling"
+       default y
+       help
+         This option enables CPU frequency scaling to happen at module level on Intel
+         processor based systems. This is useful on systems that have more than one
+         cpu per module.
+
+         if in doubt, say N
+
 config CPU_FREQ_STAT
 	tristate "CPU frequency translation statistics"
 	default y
-- 
1.9.1

